### PR TITLE
Cxf soap example 2 13 x

### PR DESCRIPTION
**Pull request by llowinge**
Note that the `main` branch points at the latest stable Camel Quarkus release.
Pull requests should be generally send against the `camel-quarkus-main` branch pointing at the current Camel Quarkus SNAPSHOT.

### CODE REVIEW

**Comment by llowinge** [comment]
@jamesnetherton FYI

File: cxf-soap/README.adoc [hunk 1]
```diff
@@ -0,0 +1,204 @@
+= Camel Quarkus CXF SOAP example
+:cq-example-description: An example that shows how to use Camel master component.
```
Comment (ppalaga) [inline cxf-soap/README.adoc:2]:
```suggestion
:cq-example-description: An example that shows how to use Camel CXF SOAP component.
```

Comment (llowinge) [reply cxf-soap/README.adoc]: Thanks for catching it @ppalaga !

**Comment by llowinge** [comment]
It should be merged after 2.13.3 release, otherwise cxf plugin doesn't work.

**Pull request merged**